scripts.py

```diff
@@ -29,17 +30,25 @@ def check_mypy():
 
 
 def test():
-    subprocess.run(["python", "-m", "pytest", "-n", "auto", "--log-level=CRITICAL"], check=True)
+    test_cmd = ["python", "-m", "pytest", "-n", "auto", "--log-level=CRITICAL"]
```

Comment:
This is nice because you can pass `-n 0` to disable xdist and it'll override the earlier -n arg.